modules/gapi/include/opencv2/gapi/video.hpp

```diff
@@ -129,6 +129,63 @@ G_TYPED_KERNEL(GBackgroundSubtractor, <GMat(GMat, BackgroundSubtractorParams)>,
     }
 };
 
+/** @brief Structure for the Kalman filter's initialization parameters.*/
+
+struct GAPI_EXPORTS KalmanParams
+{
+    // initial state
+
+    //! corrected state (x(k)): x(k)=x'(k)+K(k)*(z(k)-H*x'(k))
+    Mat state;
+    //! posteriori error estimate covariance matrix (P(k)): P(k)=(I-K(k)*H)*P'(k)
+    Mat errorCov;
+
+    // dynamic system description
+
+    //! state transition matrix (A)
+    Mat transitionMatrix;
+    //! measurement matrix (H)
+    Mat measurementMatrix;
+    //! process noise covariance matrix (Q)
+    Mat processNoiseCov;
+    //! measurement noise covariance matrix (R)
+    Mat measurementNoiseCov;
+    //! control matrix (B) (Optional: not used if there's no control)
+    Mat controlMatrix;
+};
+
+void checkParams(const KalmanParams& kfParams, const GMatDesc& measurement,
+                        const bool controlCapable, const int type);
```

Comment:
indent

Reply:
https://github.com/opencv/opencv/pull/18869#discussion_r540479544

Please don't duplicate comments.

Reply:
Sorry, saw this comment but thought it is about something else..

Reply:
Done.